src/PixiEditor/ViewModels/SubViewModels/LayersViewModel.cs

```diff
@@ -171,7 +172,7 @@ public void OpacitySliderSet(double value)
     public void DuplicateLayer()
     {
         var member = Owner.DocumentManagerSubViewModel.ActiveDocument?.SelectedStructureMember;
-        if (member is not LayerViewModel layerVM)
+        if (member is not ImageLayerNodeViewModel layerVM)
```

Comment:
Checking against generic LayerViewModel would be better here, ability to duplicate any layer is desired